students/Tejal/Lesson3/mailroom.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+import sys
+import os
+import tempfile
+import time
+
+
+
+donations = {1:{'name':'John Kestrel','amount':[653772.32, 12.17, 4]},
+2:{'name':'Jeff Kingbird','amount':[877.33]},
+3:{'name':'Paul Jacobin','amount':[663.23, 43.87, 1.32]},
+4:{'name':'Mark Tanager','amount':[1663.23, 4300.87, 10432.0]},
+5:{'name':'Anna Hummingbird','amount':[4500.0, 800.0, 2350.0]},
+6:{'name':'Calvin Fannin', 'amount':[645.0]}}
```

Comment:
you should be using a list for your data structure in this lesson - please correct.